ignite/contrib/engines/common.py

```diff
@@ -262,8 +284,8 @@ def setup_any_logging(logger, logger_module, trainer, optimizers, evaluators, lo
 def _setup_logging(
     logger: BaseLogger,
     trainer: Engine,
-    optimizers: Union[Optimizer, Dict[str, Optimizer]],
-    evaluators: Union[Engine, Dict[str, Engine]],
+    optimizers: Union[Optimizer, Dict[str, Optimizer], None],
+    evaluators: Union[Engine, Dict[str, Engine], None],
```

Comment:
we usually use `Optional[Union[...]]` instead of adding `None` inside the `Union` part. Please see my comment below for the corrected signature.